tests/conftest.py

```diff
@@ -294,6 +301,30 @@ async def mlserver(
     assert not prom_files
 
 
+@pytest.fixture
+async def echo_mlserver(
```

Comment:
Instead of creating a new one, I think we could just load the echo model on a `batch_processing/conftest.py` fixture? Or even just in the new test. That way we avoid having to maintain multiple similar fixtures.

Reply:
I may not read it correct, but I understood that `mlserver` fixture was having hard-coded `sum-model` to be used. I may have missed the way to customize it though?

Reply:
Fixed